libcxx/test/std/utilities/meta/meta.unary/meta.unary.prop/reference_converts_from_temporary.pass.cpp

```diff
@@ -0,0 +1,90 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++23
+
+// These compilers don't support std::reference_converts_from_temporary yet.
+// UNSUPPORTED: android && clang-19.0, apple-clang-15, apple-clang-16, clang-18
```

Comment:
Is the `android && clang-19.0` configuration not using a "true" Clang 19? I think this configuration should be supported.

Reply:
Please check the comment above: https://github.com/llvm/llvm-project/pull/128649#discussion_r1982780610